packages/tradershub/src/features/cfd/screens/ChangePassword/ChangePassword.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Modal } from '../../../../components/Modal';
+import { Provider } from '@deriv/library';
+import { PlatformDetails } from '../../constants';
+import MT5ChangePasswordScreens from './MT5ChangePasswordScreens';
+import TradingPlatformChangePasswordScreens from './TradingPlatformChangePasswordScreens';
+
+const ChangePassword = () => {
+    const { getCFDState } = Provider.useCFDContext();
+    const platform = getCFDState('platform') ?? PlatformDetails.mt5.platform;
+    const { title } = PlatformDetails[platform];
+
+    const isDerivX = platform === PlatformDetails.dxtrade.platform;
```

Comment:
you can use CFDPlatforms instead of PlatformDetails, but it's up to you of course :)